lang/py/src/avro/io.py

```diff
@@ -38,8 +38,11 @@
 """
 import struct
 from avro import schema
+from avro import constants
+from avro.lib import timezones
```

Comment:
@abrarsheikh you probably forgot to commit this file (this import fails)
Using `pytz` and `pytz.utc` works fine.